The systemd dep is only required for determining the systemd{user,system}unitddir
--- a/meson.build.orig	2020-08-31 12:07:34.734015573 +0200
+++ b/meson.build	2020-08-31 12:07:42.170625714 +0200
@@ -93,7 +93,6 @@
   libwrap_dep = cc.find_library('libwrap')
 endif
 # systemd
-systemd_dep = dependency('systemd')
 libsystemd_dep = dependency('libsystemd')
 journald_dep = dependency('journald', required: get_option('systemd-journal'))
 systemd_multiseat_x = find_program('systemd-multi-seat-x', required: false)
